chef/cookbooks/provisioner/templates/suse/crowbar_register.erb

```diff
@@ -246,8 +206,7 @@ fi
 # Check that we can really register this node
 # -------------------------------------------
 
-nodes=$(get_nodes_allow_fail || echo "")
-if test -z "$nodes"; then
+if ! crowbarctl node list > /dev/null; then
```

Comment:
That one is wrong, and tricky. When people use crowbar_join, you can't expect crowbarctl to be installed on the node. It's actually a package that should be added to the list of things to install a few lines later (look for zypper in this script).

One solution would be to first install crowbarctl, and then do the checks like this.